scss/style.scss

```diff
@@ -2,5 +2,145 @@
 
 @import "variables";
 @import "typography";
-
 // import components here
+@font-face {
+    font-family: Raleway; 
+    src: url(src/fonts/Raleway-SemiBold.ttf); 
+   }
+* {
+  margin: 0;
+  padding: 0;
+  font-family: Raleway, sans-serif;
+}
+
+a {
+    text-decoration: none;
+    color:#FAFAFA;
+}
+body {
```

Comment:
It's not a good practice to apply styles to `body`. Use wrapper block instead.

Reply:
fixed